### PR TITLE
Runtime config

### PR DESCRIPTION
**Pull request by a-ba**
This PR bring multiple changes to ease the deployment and the configuration of shanoir in production
- it ensures that the runtime config is *effectively* applied at runtime with environment variables rather than at build-time (parameters like server-url, admin-email, ... should not be hardcoded in the images)
- it provides a facade for the most common parameters that need to be configured by the sysadmin. The environment variables are reduced to a small set (the SHANOIR_* variables), which cover the most common setups and avoid exposing the implementation details to the sysadmin.
- it adds a bootstrap script to build and deploy shanoir from scratch

Notes:
- The PR is pretty big. You should rather review the commits individually (and they all have a detailed commit message).
- I still need to update the README.md file

have a nice week-end

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -28,6 +28,15 @@ SHANOIR_PREFIX=
 SHANOIR_X_FORWARDED=generate
 
 
+# SHANOIR_CERTIFICATE: Generation of the TLS server certificate
+# 
+# Possible values:
```
Comment (jcomedouteau) [inline .env:39]: Does INRIA provide valid certificates ? Otherwise we could propose let's encrypt (via certbot) ?

Comment (a-ba) [reply .env:39]: yes, and we are specifically requested *not* to use let's encrypt

**Comment by jcomedouteau** [comment]
Quand je déploie et que j'essaie de charger les études (ou autre), j'ai l'erreur suivante. Ca a l'air d'être en rapport avec la configuration de keycloak, mais je n'ai pas réussit à trouver pourquoi.
> 2020-06-08 08:30:58.715 ERROR 1 --- [http-nio-9902-exec-4] o.k.a.rotation.AdapterTokenVerifier      : Didn't find publicKey for kid: SWzVej9NjoUaXoBjyLWQKKjG5FDYNFEj_X1VVxBjs3g
> 2020-06-08 08:30:58.715 ERROR 1 --- [http-nio-9902-exec-4] o.k.a.BearerTokenRequestAuthenticator    : Failed to verify token
> 
> org.keycloak.common.VerificationException: Didn't find publicKey for specified kid
> 	at org.keycloak.adapters.rotation.AdapterTokenVerifier.getPublicKey(AdapterTokenVerifier.java:124)
> 	at org.keycloak.adapters.rotation.AdapterTokenVerifier.createVerifier(AdapterTokenVerifier.java:111)
> 	at org.keycloak.adapters.rotation.AdapterTokenVerifier.verifyToken(AdapterTokenVerifier.java:47)
> 	at org.keycloak.adapters.BearerTokenRequestAuthenticator.authenticateToken(BearerTokenRequestAuthenticator.java:103)
> 	at org.keycloak.adapters.BearerTokenRequestAuthenticator.authenticate(BearerTokenRequestAuthenticator.java:88)
> 	at org.keycloak.adapters.RequestAuthenticator.authenticate(RequestAuthenticator.java:68)
> 	at org.keycloak.adapters.springsecurity.filter.KeycloakAuthenticationProcessingFilter.attemptAuthentication(KeycloakAuthenticationProcessingFilter.java:149)

**Comment by a-ba** [comment]
très probablement un problème avec le certificat X.509 qui n'est pas reconnu. Tu peux vérifier s'il est bien généré dans le volume certificate-share-data

**Comment by jcomedouteau** [comment]
Dans le volume, j'ai 
cacerts
shanoir-ng-nginx.crt
shanoir-ng-nginx.key

Les trois fichiers crées ce matin.

**Comment by michaelkain** [comment]
Hi Anthony,
I reviewed your PR this morning and this is really great work!! Now it looks really wired for production and it should be easy, even for us, to run multiple instance on our machines!
First, I am ready to accept your PR, as soon as the unit tests work and the build runs, as otherwise it will damage the develop branch. I hope this is okay for you. And I only have one comment concerning the named volumes currently used, that are maybe not really adapted for prod usage. I already recognized a really big performance difference when testing the migration scripts locally. Do you have any idea or comment on that?
Thank you for your great contribution and the clear readme.md now, have a nice day, Michael

**Comment by jcomedouteau** [comment]
@a-ba 
Après discussion avec Michael, l'utilisation des placeholder dans les application.yml ne semble pas forcément la meilleure solution. Notamment au niveau des tests unitaires, ils sont du coup dépendants des variables d'environnement qui sont définies uniquement lors du déploiement de docker-compose.
On peut en discuter demain pendant la réunion si tu veux bien.

**Comment by a-ba** [comment]
I managed to run the tests, just be setting the missing env vars with dummy values.

eg:
```
3de927d25f48:/src/shanoir-ng-parent$ SHANOIR_URL_HOST=localhost SHANOIR_URL_SCHEME=http SHANOIR_PREFIX= SHANOIR_ADMIN_EMAIL='nobody@inria.fr' mvn test 2>&1 | grep "Tests run:"
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 17.512 sec - in org.shanoir.ng.accountrequest.AccountRequestApiSecurityTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.26 sec - in org.shanoir.ng.accountrequest.AccountRequestApiControllerTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.074 sec - in org.shanoir.ng.role.RoleServiceTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.277 sec - in org.shanoir.ng.role.RoleApiControllerTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.65 sec - in org.shanoir.ng.role.RoleRepositoryTest
Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.547 sec - in org.shanoir.ng.user.UserApiControllerTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.526 sec - in org.shanoir.ng.user.UserRepositoryTest
Tests run: 19, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.706 sec - in org.shanoir.ng.user.UserServiceTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.615 sec - in org.shanoir.ng.user.UserServiceSecurityTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.02 sec - in org.shanoir.ng.email.EmailServiceTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 sec - in org.shanoir.ng.utils.PasswordUtilsTest
Tests run: 62, Failures: 0, Errors: 0, Skipped: 0
Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 sec - in org.shanoir.anonymization.anonymization.AnonymizationTest
Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 18.376 sec - in org.shanoir.ng.timepoint.TimepointMapperTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.166 sec - in org.shanoir.ng.acquisitionequipment.AcquisitionEquipmentServiceTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.412 sec - in org.shanoir.ng.acquisitionequipment.AcquisitionEquipmentMapperTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.335 sec - in org.shanoir.ng.acquisitionequipment.AcquisitionEquipmentSecurityTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.044 sec - in org.shanoir.ng.acquisitionequipment.AcquisitionEquipmentApiControllerTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.998 sec - in org.shanoir.ng.coil.CoilRepositoryTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.026 sec - in org.shanoir.ng.coil.CoilMapperTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.557 sec - in org.shanoir.ng.coil.CoilApiControllerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.162 sec - in org.shanoir.ng.coil.CoilSecurityTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.015 sec - in org.shanoir.ng.coil.CoilServiceTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.505 sec - in org.shanoir.ng.center.CenterMapperTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.64 sec - in org.shanoir.ng.center.CenterRepositoryTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.971 sec - in org.shanoir.ng.center.CenterSecurityTest
Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.025 sec - in org.shanoir.ng.center.CenterServiceTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.161 sec - in org.shanoir.ng.center.CenterApiControllerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.122 sec - in org.shanoir.ng.study.service.StudyBIDSServiceImplTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.099 sec - in org.shanoir.ng.study.StudyServiceSecurityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.844 sec - in org.shanoir.ng.study.StudyApiSecurityTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.047 sec - in org.shanoir.ng.study.StudyServiceTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.656 sec - in org.shanoir.ng.study.StudyMapperTest
Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.177 sec - in org.shanoir.ng.study.StudyApiControllerTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.459 sec - in org.shanoir.ng.study.StudyRepositoryTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.684 sec - in org.shanoir.ng.study.StudySecurityTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.026 sec - in org.shanoir.ng.manufacturermodel.ManufacturerModelMapperTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.889 sec - in org.shanoir.ng.manufacturermodel.ManufacturerModelApiControllerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.002 sec - in org.shanoir.ng.manufacturermodel.ManufacturerModelServiceTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.121 sec - in org.shanoir.ng.manufacturermodel.ManufacturerModelRepositoryTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.007 sec - in org.shanoir.ng.manufacturermodel.ManufacturerServiceTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.096 sec - in org.shanoir.ng.manufacturermodel.ManufacturerSecurityTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.216 sec - in org.shanoir.ng.manufacturermodel.ManufacturerRepositoryTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.144 sec - in org.shanoir.ng.manufacturermodel.ManufacturerModelSecurityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.837 sec - in org.shanoir.ng.manufacturermodel.ManufacturerApiControllerTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.016 sec - in org.shanoir.ng.utils.UtilsTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.586 sec - in org.shanoir.ng.studycenter.StudyCenterMapperTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.01 sec - in org.shanoir.ng.subject.SubjectServiceTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.512 sec - in org.shanoir.ng.subject.SubjectApiControllerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.355 sec - in org.shanoir.ng.subject.SubjectRepositoryTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.634 sec - in org.shanoir.ng.subject.SubjectServiceSecurityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.183 sec - in org.shanoir.ng.subject.SubjectApiSecurityTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.024 sec - in org.shanoir.ng.groupofsubjects.ExperimentalGroupOfSubjectsMapperTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.19 sec - in org.shanoir.ng.subjectstudy.SubjectStudyRepositoryTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.016 sec - in org.shanoir.ng.subjectstudy.SubjectStudyMapperTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.003 sec - in org.shanoir.ng.bids.service.BidsDeserializerTest
Tests run: 186, Failures: 0, Errors: 0, Skipped: 0
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 13.323 sec - in org.shanoir.ng.examination.ExaminationRepositoryTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.12 sec - in org.shanoir.ng.examination.ExaminationMapperTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.643 sec - in org.shanoir.ng.examination.ExaminationServiceSecurityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 13.306 sec - in org.shanoir.ng.examination.ExaminationApiSecurityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.421 sec - in org.shanoir.ng.examination.ExaminationServiceTest
Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.375 sec - in org.shanoir.ng.examination.ExaminationApiControllerTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.953 sec - in org.shanoir.ng.studycard.StudyCardRepositoryTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.15 sec - in org.shanoir.ng.studycard.StudyCardServiceTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.999 sec - in org.shanoir.ng.studycard.StudyCardSecurityTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.188 sec - in org.shanoir.ng.studycard.StudyCardApiControllerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 15.515 sec - in org.shanoir.ng.studycard.StudyCardServiceSecurityTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.972 sec - in org.shanoir.ng.datasetacquisition.service.ImporterServiceTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 12.163 sec - in org.shanoir.ng.datasetacquisition.DatasetAcquisitionServiceSecurityTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.01 sec - in org.shanoir.ng.datasetacquisition.DatasetAcquisitionMapperTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 13.489 sec - in org.shanoir.ng.datasetacquisition.DatasetAcquisitionApiSecurityTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.452 sec - in org.shanoir.ng.datasetacquisition.DatasetAcquisitionRepositoryTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.746 sec - in org.shanoir.ng.datasetacquisition.DatasetAcquisitionApiControllerTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 11.323 sec - in org.shanoir.ng.dataset.DatasetRepoTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.965 sec - in org.shanoir.ng.dataset.DatasetApiControllerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 14.056 sec - in org.shanoir.ng.dataset.DatasetApiSecurityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 13.891 sec - in org.shanoir.ng.dataset.DatasetServiceSecurityTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.313 sec - in org.shanoir.ng.dataset.DatasetRepositoryTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.029 sec - in org.shanoir.ng.dataset.DatasetMapperTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 14.516 sec - in org.shanoir.ng.studyuser.StudyUserProcessCommandTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 14.903 sec - in org.shanoir.ng.tasks.ShanoirEventServiceTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.121 sec - in org.shanoir.ng.importer.strategies.protocol.MrProtocolStrategyTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 7.013 sec - in org.shanoir.ng.exporter.service.BidsServiceTest
Tests run: 94, Failures: 0, Errors: 0, Skipped: 0
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.23 sec - in org.shanoir.ng.exchange.imports.subject.IdentifierCalculatorTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.07 sec - in org.shanoir.ng.exchange.imports.dicom.DicomDirGeneratorServiceTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.295 sec - in org.shanoir.ng.exchange.model.ExchangeModelTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.536 s - in org.shanoir.ng.importer.eeg.BrainVisionReaderTest
[INFO] Tests run: 14, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 9.21 s - in org.shanoir.ng.importer.ImporterApiControllerTest
[INFO] Tests run: 15, Failures: 0, Errors: 0, Skipped: 0
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.823 sec - in org.shanoir.ng.preclinical.pathologies.PathologyApiControllerTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.17 sec - in org.shanoir.ng.preclinical.pathologies.PathologyServiceTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.277 sec - in org.shanoir.ng.preclinical.pathologies.pathology_models.PathologyModelRepositoryTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.953 sec - in org.shanoir.ng.preclinical.pathologies.pathology_models.PathologyModelApiControllerTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.034 sec - in org.shanoir.ng.preclinical.pathologies.pathology_models.PathologyModelServiceTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.25 sec - in org.shanoir.ng.preclinical.pathologies.subject_pathologies.SubjectPathologyRepositoryTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.618 sec - in org.shanoir.ng.preclinical.pathologies.subject_pathologies.SubjectPathologyApiControllerTest
Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.051 sec - in org.shanoir.ng.preclinical.pathologies.subject_pathologies.SubjectPathologyServiceTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.959 sec - in org.shanoir.ng.preclinical.pathologies.PathologyRepositoryTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.492 sec - in org.shanoir.ng.preclinical.subjects.AnimalSubjectRepositoryTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.327 sec - in org.shanoir.ng.preclinical.subjects.AnimalSubjectApiControllerTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.022 sec - in org.shanoir.ng.preclinical.subjects.AnimalSubjectServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.836 sec - in org.shanoir.ng.preclinical.examination_extradata.ExtraDataRepositoryTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.026 sec - in org.shanoir.ng.preclinical.examination_extradata.ExtraDataServiceTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.277 sec - in org.shanoir.ng.preclinical.examination_extradata.ExtraDataApiControllerTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.167 sec - in org.shanoir.ng.preclinical.anesthetics.anesthetic_ingredients.AnestheticIngredientApiControllerTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.669 sec - in org.shanoir.ng.preclinical.anesthetics.anesthetic_ingredients.AnestheticIngredientRepositoryTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.022 sec - in org.shanoir.ng.preclinical.anesthetics.anesthetic_ingredients.AnestheticIngredientServiceTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.073 sec - in org.shanoir.ng.preclinical.anesthetics.examination_anesthetics.ExaminationAnestheticApiControllerTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.519 sec - in org.shanoir.ng.preclinical.anesthetics.examination_anesthetics.ExaminationAnestheticRepositoryTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.016 sec - in org.shanoir.ng.preclinical.anesthetics.examination_anesthetics.ExaminationAnestheticServiceTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.989 sec - in org.shanoir.ng.preclinical.anesthetics.AnestheticApiControllerTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.015 sec - in org.shanoir.ng.preclinical.anesthetics.AnestheticServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.822 sec - in org.shanoir.ng.preclinical.anesthetics.AnestheticRepositoryTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.019 sec - in org.shanoir.ng.preclinical.therapies.TherapyServiceTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.173 sec - in org.shanoir.ng.preclinical.therapies.subject_therapies.SubjectTherapyApiControllerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.338 sec - in org.shanoir.ng.preclinical.therapies.subject_therapies.SubjectTherapyRepositoryTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.017 sec - in org.shanoir.ng.preclinical.therapies.subject_therapies.SubjectTherapyServiceTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.893 sec - in org.shanoir.ng.preclinical.therapies.TherapyApiControllerTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.391 sec - in org.shanoir.ng.preclinical.therapies.TherapyRepositoryTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.26 sec - in org.shanoir.ng.preclinical.references.ReferencesRepositoryTest
Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.008 sec - in org.shanoir.ng.preclinical.references.ReferenceServiceTest
Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.047 sec - in org.shanoir.ng.preclinical.references.ReferencesApiControllerTest
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.849 sec - in org.shanoir.ng.preclinical.bruker.BrukerApiControllerTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.072 sec - in org.shanoir.ng.preclinical.contrast_agents.ContrastAgentApiControllerTest
Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.021 sec - in org.shanoir.ng.preclinical.contrast_agents.ContrastAgentServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.318 sec - in org.shanoir.ng.preclinical.contrast_agents.ContrastAgentRepositoryTest
Tests run: 202, Failures: 0, Errors: 0, Skipped: 0
```

**Comment by jcomedouteau** [comment]
I'm not sure that it's a good solution for us to have these environnement variables permanently, what do you think of it @michaelkain ?

**Pull request merged**